libraries/shared/transformer/storage_transformer.go

```diff
@@ -25,6 +25,8 @@ import (
 type StorageTransformer interface {
 	Execute(diff types.PersistedDiff) error
 	KeccakContractAddress() common.Hash
+	GetStorageKeysLookup() interface{}
```

Comment:
I'd love some feedback on this. Because of the way the project is structured, if we have this return a `storage.KeysLookup` we end up having a circular dependency because the `storage` package imports this `transformer` package. 

I'm wondering if it makes sense to have this `StorageTransformer` interface here in it's own package, or if we could potentially move it to the `storage` package where the concrete Transformer is being implemented. I'm thinking of a similar change for the event transformer interface. 🤔

Reply:
💯 to moving it, and returning a more specific type

Reply:
👍 updated in b544c91ff214a787ec509b8ba75fb7eaf2df29e8

will likely do the same for the EventTransformer and ContractTransformer interfaces in another PR.

Reply:
Actually, I am going to remove that change from this PR - as I was testing this out, I realized that it will affect the way that the [plugin is built ](https://github.com/makerdao/vulcanizedb/blob/staging/pkg/plugin/writer/writer.go#L60). So I think I'll hold off on that so that this PR isn't too big, and create a story for dealing with this for storage, event and contract transformers in the future.